www/source/partials/docs/_glossary-keys.html.md.erb

```diff
@@ -1,14 +1,29 @@
 # <a name="glossary-keys" id="glossary-keys" data-magellan-target="glossary-keys">Keys</a>
 
-Habitat has strong cryptography built into both Habitat Builder and the Supervisor. This means there are several different kinds of keys.
+Habitat has strong cryptography built into Habitat Builder, the Supervisor, and the `hab` CLI commands. This means there are several different kinds of keys.
 
-## Origin Keys
+## Origin Key Pairs
 
-As described in the [packages documentation](/docs/glossary/#glossary-packages), every package in Habitat belongs to an origin and is cryptographically signed with that origin's private key.
+Every Habitat artifact belongs to an [origin](/docs/glossary/#glossary-artifacts) and is cryptographically signed with that origin's private key.
+Habitat requires both the public and the private key for producing and consuming artifacts.
+On the production side, Builder---both the service and the Depot---use the origin private key to sign and verify packages as they move from your development environment to Builder. On the consumption side, the Supervisors require the public key to verify the compiled artifacts.
```

Comment:
```suggestion
On the production side, Builder---both the service and the Depot---use the origin private key to sign and verify packages as they move from your development environment to Builder. On the consumption side, Habitat requires the public key to verify the compiled artifacts during installation.
```
The supervisor doesn't do anything related to artifact verification; that's a standard part of the install procedure, which is identical whether initiated by the user directly (as in `hab pkg install`) or implicitly through a supervisor-oriented command such as `hab svc load`.